coral-incremental/src/main/java/com/linkedin/coral/incremental/IncrementalTransformerResults.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.incremental;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.calcite.rel.RelNode;
+
+
+public class IncrementalTransformerResults {
+
+  private RelNode incrementalRelNode;
+  private RelNode refreshRelNode;
+  private Map<String, RelNode> intermediateQueryRelNodes;
+  private List<String> intermediateOrderings;
+
+  public IncrementalTransformerResults() {
+    incrementalRelNode = null;
+    refreshRelNode = null;
```

Comment:
Should it be a list of refresh relnodes? We can remove from current PR till we converge on the best representation.

Reply:
Yes, this should be a list of refresh RelNodes—one for each aggregate query. Will remove for now.